PowerShell/JumpCloud Commands Gallery/Mac Commands/Mac - Set ReleaseChannel In JumpCloud Password Manager.md

```diff
@@ -0,0 +1,44 @@
+#### Name
+
+Mac - Set JumpCloud Password Manager Release Channel | v1.0 JCCG
+
+#### commandType
+
+mac
+
+#### Command
+
+```
+#!/bin/bash
+
+# Set $RELEASE_CHANNEL to beta OR dogfood OR public ON LINE 15 depending on your desired release channel
+RELEASE_CHANNEL="public"
```

Comment:
We can't expect admins to do the right thing, it would probably be a good idea to add a check to ensure that `$RELEASE_CHANNEL` is one of 'public','dogfood','beta'